runtime/runtime-params-estimator/src/config.rs

```diff
@@ -37,6 +26,6 @@ pub struct Config {
     pub metric: GasMetric,
     /// VMKind used
     pub vm_kind: VMKind,
-    /// When non-none, only the specified metrics will be measured.
-    pub metrics_to_measure: Option<Vec<String>>,
+    /// When non-none, only the specified costs will be measured.
+    pub costs_to_measure: Option<Vec<String>>,
```

Comment:
Not sure if i mentioned this alraedy, but it makes sense to rename the cli flag to jsut `--costs` as well.